netty/src/main/java/io/grpc/netty/ProtocolNegotiators.java

```diff
@@ -73,7 +76,7 @@ private ProtocolNegotiators() {
   /**
    * Create a TLS handler for HTTP/2 capable of using ALPN/NPN.
    */
-  public static ChannelHandler serverTls(SSLEngine sslEngine) {
+  public static ChannelHandler serverTls(SSLEngine sslEngine, final ChannelHandler grpcHandler) {
```

Comment:
Could we go back to just doing `pipeline.remove()` for the moment? I don't mind the replace, but see it as an unnecessary change right now, and would prefer not to make unnecessary changes at this point.